src/bundles.sh

```diff
@@ -0,0 +1,18 @@
+export CONDUCTR_IP=$(hostname -i)
+
+conduct load conductr-haproxy
+conduct run conductr-haproxy --scale $NODE_COUNT
+
+sudo service haproxy restart
+
+sleep 5
```

Comment:
Sorry to be a pain but it'd be great to reproduce this issue requiring a sleep. If there is a race condition then it could also occur in prod.